src/include/connection.h

```diff
@@ -213,6 +213,7 @@ struct __wt_connection_impl {
 
     WT_LSN *debug_ckpt;      /* Debug mode checkpoint LSNs. */
     uint32_t debug_ckpt_cnt; /* Checkpoint retention number */
+    bool debug_slow_ckpt;    /* Debug mode slow checkpoints */
```

Comment:
It occurs to me now that there really isn't a need to expand the size of the connection structure for this work. Can you, instead, define and use a bit flag in the connection instead? I suggest calling it `WT_CONN_DEBUG_SLOW_CKPT`. @luke-pearson should be able to help with how to do so using the automatic flag generation/renumbering. See `WT_CONN_LEAK_MEMORY` for similar boolean style usage.